rdsn/thirdparty/CMakeLists.txt

```diff
@@ -409,3 +409,13 @@ ExternalProject_Add(http-parser
         INSTALL_COMMAND cp -R http_parser.h ${TP_OUTPUT}/include/nodejs/http_parser.h
         BUILD_IN_SOURCE 1
         )
+
+# ExternalProject_Add(python
+#         URL ${OSS_URL_PREFIX}/Python-2.7.18.tgz
+#         https://www.python.org/ftp/python/2.7.18/Python-2.7.18.tgz
```

Comment:
python3 is installed on each Dockerfile, no need to installed again. See
https://github.com/apache/incubator-pegasus/blob/master/docker/pegasus-build-env/centos7/Dockerfile#L28

Reply:
python3 . ok .
I'll try again.